flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvRowDataSerDeSchemaTest.java

```diff
@@ -84,11 +84,11 @@ void testSerializeDeserialize() throws Exception {
         testNullableField(DOUBLE(), "0.33333333332", 0.33333333332d);
         testNullableField(
                 DECIMAL(38, 25),
-                "\"1234.0000000000000000000000001\"",
+                "1234.0000000000000000000000001",
```

Comment:
where does this change come from? It seems significant...

Reply:
came with jackson-format-csv (jackson-format-text) 2.16.0
I their GH it is mentioned as a bug fix
https://github.com/FasterXML/jackson-dataformats-text/issues/438